recipe/meta.yaml

```diff
@@ -76,78 +72,115 @@ requirements:
     - {{ mpi }}  # [mpi != 'nompi']
     - zlib
 
-test:
-  requires:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - {{ compiler('fortran') }}  # [unix]
-    - pkg-config  # [unix]
-
-  files:
-    - h5_cmprss.c
-    - h5tutr_cmprss.cpp
-    - h5_cmprss.f90
-    - compound_fortran2003.f90
+outputs:
+  - name: hdf5
+    build:
+      # run_exports:
+      #   - {{ pin_subpackage('hdf5', max_pin='x.x.x') }} {{ build_pin }}
```

Comment:
A run_export should be declared

Reply:
Ah. I removed this to test it. I cannot build or re-render with a `run_exports` in this line. Not sure what is going on. I guess it is the non-standard build string.

Reply:
Things worked now. Not sure if I had a <tab> somwhere or if the new conda-build in my env fixed it.